### PR TITLE
[SPARTA-654] [PLUGINS] Rabbitmq version updated to 0.3

### PR DESCRIPTION
**Pull request by compae**

### CODE REVIEW

File: web/src/data-templates/input.json [hunk 1]
```diff
@@ -346,67 +346,45 @@
...
+        "fields": [
+          {
+            "propertyId": "rabbitmqPropertyKey",
+            "propertyName": "_RABBITMQ_PROPERTY_KEY_",
+            "propertyType": "text",
+            "regexp": "",
+            "default": "",
+            "hidden": false,
+            "required": false,
+            "qa": "fragment-details-kafka-rabbitmqPropertyKey"
```
Comment (danielcsant) [inline web/src/data-templates/input.json]: "qa": "fragment-details-rabitmq-rabbitmqPropertyKey"

File: web/src/data-templates/input.json [hunk 2]
```diff
@@ -346,67 +346,45 @@
...
+          },
+          {
+            "propertyId": "rabbitmqPropertyValue",
+            "propertyName": "_RABBITMQ_PROPERTY_VALUE_",
+            "propertyType": "text",
+            "regexp": "",
+            "default": "",
+            "hidden": false,
+            "required": false,
+            "qa": "fragment-details-kafka-rabbitmqPropertyValue"
```
Comment (danielcsant) [inline web/src/data-templates/input.json]: "qa": "fragment-details-rabbitmq-rabbitmqPropertyValue"

File: plugins/src/main/scala/com/stratio/sparta/plugin/input/rabbitmq/RabbitMQInput.scala [hunk 3]
```diff
@@ -13,61 +13,56 @@
...
     }
   }
+
+  private def getRabbitMQProperties: Map[String, String] =
+    Try(properties.getMapFromJsoneyString(RabbitmqProperties))
+      .getOrElse(Seq.empty[Map[String, String]])
+      .map(c =>
+        (c.get(RabbitmqPropertyKey) match {
+          case Some(value) => value.toString
+          case None => throw new IllegalStateException(s"The field $RabbitmqPropertyKey is mandatory")
```
Comment (danielcsant) [inline plugins/src/main/scala/com/stratio/sparta/plugin/input/rabbitmq/RabbitMQInput.scala]: Should be IllegalArgumentException

File: plugins/src/main/scala/com/stratio/sparta/plugin/input/rabbitmq/RabbitMQInput.scala [hunk 4]
```diff
@@ -13,61 +13,56 @@
...
+        },
+          c.get(RabbitmqPropertyValue) match {
+            case Some(value) => value.toString
+            case None => throw new IllegalStateException(s"The field $RabbitmqPropertyValue is mandatory")
```
Comment (danielcsant) [inline plugins/src/main/scala/com/stratio/sparta/plugin/input/rabbitmq/RabbitMQInput.scala]: Should be IllegalArgumentException

**Comment by danielcsant** [comment]
Update documentation

https://stratio.atlassian.net/wiki/display/SPARKTA/Inputs#Inputs-RabbitMQ

**Comment by compae** [comment]
All changes done!! Documentation updated!!

**Comment by danielcsant** [comment]
LGTM

**Comment by danielcsant** [comment]
:shipit:

**Comment by stratiocommit** [comment]
Comment accepted. Will treat it ASAP on [Jenkins](http://jenkins.stratio.com/job/Sparta/job/CD/2982/)

**Pull request merged**